DECMOS/decmos.py

```diff
@@ -47,7 +47,8 @@ def main(input_dir, dataset_dir, score_file):
         lpb_sig, mic_sig, enh_sig = read_and_process_audio_files(
             lpb_path, mic_path, clip_path)
 
-        clip_scores = get_score(lpb_sig, mic_sig, enh_sig)
+        clip_scores = get_score(lpb_sig, mic_sig, enh_sig,
+                                get_clip_scenario(os.path.basename(clip_path)))
```

Comment:
Long term, the scenario shouldn't be based on file name

Reply:
I think taking CSV-s as input would be the best solution long-term